rust/kernel/net/skbuff.rs

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Definitors for the struct [`crate::bindings::sk_buff`] memory handlers.
+
+use core::{ops::Drop, ptr};
+
+use crate::bindings;
+use crate::types::SavedAsPointer;
+
+/// Wraps the kernel's `struct sk_buff`.
+///
+/// # Invariants
+///
+/// The pointer `Self::ptr` is non-null and valid.
+#[repr(transparent)]
+pub struct SkBuff {
+    ptr: *const bindings::sk_buff,
+}
+
+impl SkBuff {
+    /// Binding to `bindings::skb_clone_tx_timestamp`.
+    ///
+    /// Noop if `CONFIG_NETWORK_PHY_TIMESTAMPING` is not enabled.
```

Comment:
Given the docs are separate (i.e. we have one doc for the config option enabled, and one with it disabled), I would not repeat the docs.

Also, we should explain what the function does, instead of (only) saying it is a binding, because the user of the documentation then needs to go and find the documentation for that function.

```suggestion
    /// Binding to `bindings::skb_clone_tx_timestamp`.
```